src/core/iter.rs

```diff
@@ -94,3 +96,20 @@ impl<'a, D: DagIterable> Iterator for PostOrderIter<'a, D> {
         }
     }
 }
+
+/// Convert the given iterator over [`Term`]s into an iterator over the contained `Witness` values.
```

Comment:
In 17ace32625de33b048b864573ba2929fddb861bf:

The iterator is over `Witness`es not `Term`s.

Reply:
Doesn't the text say that an iterator over `Term` is converted into an iterator over `Witness`? Maybe, I should reword it.

Reply:
Oh, yes, the text is correct, I just misread the function signature. Never mind.